sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -4,7 +4,7 @@
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <ProjectReference Include="..\..\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Do we have an issue to revert this after Core release?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/49825